mflowgen/Tile_MemCore/constraints/common.tcl

```diff
@@ -15,7 +15,7 @@ set clock_net  clk
 set clock_name ideal_clock
 
 create_clock -name ${clock_name} \
-             -period ${dc_clock_period} \
+             -period ${clock_period} \
```

Comment:
Seems like this would cause a compatibility issue if we ever switched back to dc. Should we change the dc node's designer interface to use clock_period instead of dc_clock_period?

Reply:
This will require changing all of the flows that use `dc_*` variables ... we can do this at the end once everything is ported to Genus.